src/az390.java

```diff
@@ -10598,6 +10601,9 @@ private void gen_lit_size(int size){
 				){			
 			lit_gen[cur_lit] = 1;
 			lit_esd[cur_lit] = esd_base[cur_esd]; // RPI 457
+                             if   (size == 1 && loc_ctr != (loc_ctr/2*2)){
+                                  loc_ctr = loc_ctr + 1;  // issue #327 force lits on even addr for ref by halfword offset counts
+                             }
```

Comment:
Suggestion: Shift code left so that it aligns with existing code.

I make no claim about understandiing the az390 logic, but I do have a question about this modification. Why are you only checking for size 1? Why not any odd size?